web/ui/two-steps-page.svelte

```diff
@@ -23,18 +24,26 @@
   }
 
   onMount(() => {
+    window.addEventListener('keydown', toggleSide)
```

Comment:
I think we can move `keycode === 'Escape'` code here.

But we always should check `document.active` in any hot key listener. User focus can be in text field or other control, where `Esc` should have different meaning.

Reply:
See `keyux` source code for details https://github.com/ai/keyux/blob/main/hotkey.js